notebooks/official/pipelines/google_cloud_pipeline_components_automl_images.ipynb

```diff
@@ -653,11 +605,11 @@
       "source": [
         "Click on the generated link to see your run in the Cloud Console.\n",
         "\n",
-        "<!-- It should look something like this as it is running:\n",
+        "<!-- It should look something like this as it's running:\n",
         "\n",
         "<a href=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" target=\"_blank\"><img src=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" width=\"40%\"/></a> -->\n",
         "\n",
-        "In the UI, many of the pipeline DAG nodes will expand or collapse when you click on them. Here is a partially-expanded view of the DAG (click image to see larger version).\n",
+        "In the UI, many of the pipeline DAG nodes expand or collapse when you click on them. Here is a partially-expanded view of the DAG (click image to see larger version).\n",
```

Comment:
- Is "UI" referencing the "Google Cloud console"? If so, change "UI" to"Google Cloud console"

- change "Here is" to "Here's"